WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -45,6 +46,8 @@ public class CoinJoinClient
 		RoundStateUpdater roundStatusUpdater,
 		string coordinatorIdentifier,
 		LiquidityClueProvider liquidityClueProvider,
+		int bestHeight,
```

Comment:
Instead of `int bestHeight`, we could have the `WasabiBackendStatusProvider` as a dependency for `CoinJoinClient` to get more up-to-date blockheight.

But IMHO that dependency is a bit too much just to have a simple check.